doc/markdown/user/implementation/library/Usage.md

```diff
@@ -8,15 +8,15 @@ Accelerator Executable Functions
 
 Functions that should be executable on an accelerator have to be annotated with the execution domain (one of `ALPAKA_FN_HOST`, `ALPAKA_FN_ACC` and `ALPAKA_FN_HOST_ACC`).
```

Comment:
Not sure if important, but maybe worth clarifying that here and later by "function" we actually mean both functions and function templates and usually actually the latter with at least an accelerator type as a template parameter.

Reply:
Actually we only mean `functors`, all the time.

Reply:
Do we? I thought only a kernel has to be a functor and internally it could also call normal functions, just properly annotated.

Isn't it better to link https://en.cppreference.com/w/cpp/utility/functional instead of the Wikipedia?

Reply:
I explicitly wanted to avoid the usage of the word functor because it has different meanings in different contexts.
This paragraph should be the abstract explanation for functions, function templates and function objects containing one of those.

Reply:
Okay with me :)